src/vaadin-grid.html

```diff
@@ -566,6 +579,7 @@
         return cell;
       }
 
+      /** @private */
       _updateRow(row, columns, section, isColumnRow, noNotify) {
```

Comment:
Called by row details mixin, should be `@protected`?

Reply:
Done.